osu.Game/Rulesets/UI/Scrolling/DrawableScrollingRuleset.cs

```diff
@@ -114,7 +114,7 @@ private void load()
                     break;
             }
 
-            double lastObjectTime = Objects.LastOrDefault()?.GetEndTime() ?? double.MaxValue;
+            double lastObjectTime = Beatmap.HitObjects.Any() ? Beatmap.GetLastObjectTime() : 0;
```

Comment:
Behaviour has changed. Prior to this change, the fallback value was `double.MaxValue`, but is now 0.

```suggestion
            double lastObjectTime = Beatmap.HitObjects.Any() ? Beatmap.GetLastObjectTime() : double.MaxValue;
```